api/question/schema_query.py

```diff
@@ -39,6 +45,33 @@ def get_room_response(self, unique_responses, question_id, question_type):
             response.append(room_response)
         return response
 
+    def resolve_filter_rooms_by_responses(self, info, lower_limit, upper_limit):
+        query = Room.query
+        rooms = query.all()
+        response_query = ResponseModel.query
+        filtered_rooms_responses = []
+        for room in rooms:
+            room_responses = response_query.filter_by(room_id=room.id)
```

Comment:
What if you just add .count at the end of this line and have only one line for total responses here?

Reply:
@Bonifase I did not include the count here because I also needed the responses themselves to calculate the average rating. If I were to add count there I would still have to add another line for getting all the responses.